README.md

```diff
@@ -1,12 +1,14 @@
 # AWS JWT Verify
 
-**Node.js** library for **verifying** JWTs signed by **Amazon Cognito**, and any **OIDC-compatible IDP** that signs JWTs with **RS256** / **RS384** / **RS512**.
+**JavaScript** library for **verifying** JWTs signed by **Amazon Cognito**, and any **OIDC-compatible IDP** that signs JWTs with **RS256** / **RS384** / **RS512**.
 
 ## Installation
 
 `npm install aws-jwt-verify`
 
-Note: this library can be used with Node.js 14 or higher. If used with TypeScript, TypeScript 4 or higher is required.
+This library can be used with Node.js 14 or higher. If used with TypeScript, TypeScript 4 or higher is required.
+
+This library can also be used in Web browsers (modern browsers that support e.g. `SubtleCrypto`).
```

Comment:
As discussed , call out major browser versions that are supported (i.e. support `SubtleCrypto`)

Reply:
Looking on MDN:
- [SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) importKey and verify were supported in Chrome 37, Edge 12, Firefox 34, and Safari 11
- [TextEncoder](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder) was supported in Chrome 38, Edge 79, Firefox 18, and Safari 10.1
- [fetch](https://developer.mozilla.org/en-US/docs/Web/API/fetch) was supported in Chrome 42, Edge 14, Firefox 39, and Safari 10.1

Of these, Edge 79 (the first Chromium version, release in Jan 2020) is the most recent on the list.  In fact, only IE11 wouldn't natively support our library.  I'm tempted just to strike the "modern browser" parenthetical.

Reply:
Fair enough

Reply:
@hakanson , I assume you're updating the readme, correct?

Reply:
just committed that change